### PR TITLE
Documentation to run møte on Py3

### PR DESCRIPTION
**Pull request by nasirhm**
Modified Documentation for Python 3
Usage of Pipenv to facilitate with the creation of Virtualenv and installation of packages inside it.

Fixes: #134 

Dependent on PR: #135

### CODE REVIEW

**Comment by t0xic0der** [comment]
You are right. #135 needs to be merged to fully realize the changes made here.

I followed the instructions provided here for the `pull/135` branch and found them to be working just fine.

Good work, @nasirhm.

**Review by t0xic0der — APPROVED** [review]
LGTM. Followed the instructions provided here and it works just how it is supposed to.

**Comment by nasirhm** [comment]
> LGTM. Followed the instructions provided here and it works just how it is supposed to.

Thank you for the review, one thing to take care of in the Development experience in the future would be to include a sample `config.json` related file in order to make `memcached` an Optional dependency and ensuring it doesn't cause any errors in while keeping the usage of `memcached = False`
Resulted in me creating issue #140 for further research.

File: README.md [hunk 1]
```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
...
- - Python 2.7.x
+ - `memcached` (`sudo dnf install memcached`)
+ - Python 3.8.X
+ - Pipenv
 
 Installation:
- - Optional: create virtual environment
- - `pip install -r requirements.txt`
- - Install using `python setup.py develop`
+ - `pipenv install` will create a new virtual environment and install the requirements. `
```
Comment (jwflory) [inline README.md:23]: For Pipenv, I usually recommend two steps:

```sh
# Install dependencies
$ pipenv install

# Activate Python virtualenv
$ pipenv shell
```

File: README.md [hunk 2]
```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
 ### Using møte
 
 Dependencies:
- - Optional: `memcached` (`sudo dnf install memcached`)
- - Python 2.7.x
+ - `memcached` (`sudo dnf install memcached`)
+ - Python 3.8.X
+ - Pipenv
```
Comment (jwflory) [inline README.md:20]: A link to the Pipenv docs would be a good idea to add here. Also, this is packaged in Fedora too. `sudo dnf install pipenv`

File: README.md [hunk 3]
```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
...
 
 Running møte:
+ - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+   - You can do it by running the following command in command line too: `sed -i '/use_memcached = / s/False/True/' mote/config.py`
+ - Enter the Pipenv's newly created virtualenv's shell with `pipenv shell`
  - Run `python runmote.py`
 
+**Note :** Make sure `memcached` is running in the background.
```
Comment (jwflory) [inline README.md:33]: It is out of scope for this Pull Request, but maybe we can use `docker-compose` or `podman-compose` to make this easier to set up and run for new developers.

Comment (nasirhm) [reply README.md:33]: I was thinking to something similar, I don't think if we would need `docker-compose` / `podman-compose` , we can proceed with a simple `Dockerfile` too. What do you think ?

Comment (nasirhm) [reply README.md:33]: Here's an issue to keep track of it: https://github.com/fedora-infra/mote/issues/142

File: README.md (hunk 2 above)
Comment (nasirhm) [reply README.md]: Fixed: https://github.com/fedora-infra/mote/pull/141/commits/8bfdd02674b734fa256248bf2429a00fddf28a11

File: README.md (hunk 1 above)
Comment (nasirhm) [reply README.md]: Fixed: https://github.com/fedora-infra/mote/pull/141/commits/8bfdd02674b734fa256248bf2429a00fddf28a11

File: README.md [hunk 4]
```diff
@@ -15,12 +15,13 @@ møte organises and serves meetings as a drop-in replacement, without needing an
...
- - Pipenv
+ - [`memcached`](https://memcached.org) (`sudo dnf install memcached`)
+ - Python 3.8.x
+ - [Pipenv](https://docs.pipenv.org/) (`sudo dnf install pipenv`)
 
-Installation:
- - `pipenv install` will create a new virtual environment and install the requirements. `
+Virtual Environment setup:
+ - `pipenv install` will create a new virtual environment and install the requirements.
+ - `pipenv shell` will activate Python's virtual environement's shell.
```
Comment (t0xic0der) [inline README.md:24]: An ordered list maybe? (Because they need to be executed one after the other.)

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Consider the changes I suggested before implementing them. (See if they are really needed)

File: README.md (hunk 4 above)
Comment (nasirhm) [reply README.md:24]: @t0xic0der Thanks for the review, personally I feel, It's pretty intuitive with unordered lists to follow it step by step and the list representation seems simple as points, Do you think adding an ordered list would make it more simple ?

Comment (t0xic0der) [reply README.md:24]: Either could work. There are just two points anyway. :-)

**Review by jwflory — APPROVED** [review]
The docs here look good to me. Since #135 was merged, we are ready to pull this one in too.

Merging! :clapper: Thanks @nasirhm.

**Pull request merged**